instrumentation/othttp/handler_test.go

```diff
@@ -28,14 +29,17 @@ func TestHandlerBasics(t *testing.T) {
 
 	var id uint64
 	tracer := mocktrace.MockTracer{StartSpanID: &id}
+	_, meter := mockmeter.NewMeter()
 
 	h := NewHandler(
 		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			if _, err := io.WriteString(w, "hello world"); err != nil {
 				t.Fatal(err)
 			}
 		}), "test_handler",
-		WithTracer(&tracer))
+		WithTracer(&tracer),
+		WithMeter(meter),
```

Comment:
How difficult would it be to add some checks in this test to validate that the counters were incremented, something was added to the latency etc -- with validated labels?

Reply:
added